languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -22406,6 +22406,34 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- ====================================================================== -->
     <category id="COLLOQUIALISMS" name="Umgangssprache" type="register">
         <!-- TODO Make one Java-rule for context-independent, colloquial expressions and maintain a list -->
+        <rulegroup id="ZUM_STATT_FUERS" name="fürs (zum) Schneiden">
+            <rule id="ZUM_STATT_FUERS1" name="fürs (zum) Schneiden1">
```

Comment:
Subrules (also rules in rulegroups) brauchen keine ID. Wenn du ihnen trotzdem eine gibst, ist es nicht mehr wirklich eine Ruelgroup.